magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RecordAspectsService.scala

```diff
@@ -61,23 +60,24 @@ class RecordAspectsService(webHookActor: ActorRef, authClient: AuthApiClient, sy
   @ApiOperation(value = "Modify a record aspect by ID", nickname = "putById", httpMethod = "PUT", response = classOf[Aspect],
     notes = "Modifies a record aspect.  If the aspect does not yet exist on this record, it is created.")
   @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "X-Magda-TenantId", required = true, dataType = "String", paramType = "header", value = "Magda tenant id"),
```

Comment:
I think this belongs at the bottom (with X-Magda-Session) rather than the top.

Reply:
Am I missing something important here? Why is the order important?

Reply:
It's not a massive deal just makes it consistent with the other HTTP Header doc (for X-Magda-Session-Id) which is at the bottom 🤷‍♂.

Reply:
OK, will fix it.

Reply:
done